plugins/checklist/assets/javascripts/lib/rich-editor-extension.js

```diff
@@ -0,0 +1,70 @@
+/** @type {RichEditorExtension} */
+const extension = {
+  // TODO(renato): make the checkbox clickable
```

Comment:
Will these all be done in followup PR, or this one?

Reply:
I was intending to do in this one, but let's get the current state merged first – I'll follow-up all TODOs.